### DISCUSSION TITLE
Make fuzzy search work for all recipe search fields

### DISCUSSION THREAD

**Discussion by RealFoxie** [mealie-recipes/mealie#2060] [Feature Request]
For a general recipe search, a fuzzy search implementation is used (via fuse.js, e.g. in `frontend/pages/search`) This could be done in other recipe search places like `RecipeOrganizerPage.vue`.
I think this is used for the search on the pages like /recipes/categories. Currently it only does the search on the title of the recipe.
I think it would make sense to do the same kind of fuzzy search (also on the description and ingredients) on those pages.
Also for the search on the mealplan page (when adding a recipe), which now uses v-autocomplete, a custom search would make sense: Make it exact match on title of recipe first, and afterwards do the fuzzy fuse.js search. That way you could still find the recipe you mean that might mention your keyword in the description. (e.g. "chocolate" for a chocolate cake with title "dark brown cake").

**Comment by michael-genson** [comment]
Closing this as there is only one recipe search page now
